cloud/amazon/sns_topic.py

```diff
@@ -95,11 +94,15 @@ def canonicalize_endpoint(protocol, endpoint):
 
 
 
-def get_all_topics(connection):
+def get_all_topics(connection, module):
     next_token = None
     topics = []
     while True:
-        response = connection.get_all_topics(next_token)
+        try:
+            response = connection.get_all_topics(next_token)
+        except BotoServerError as e:
```

Comment:
That is interesting. I've used 'as' for exception handling all over my cloud modules which have been merged without any comments from the core team.

I don't see the point in writing a module as 2.4 compatible when we know that it'll only ever work with 2.6+ because of dependencies such as boto.